compiler/src/iree/compiler/Dialect/HAL/Transforms/TranslateExecutables.cpp

```diff
@@ -118,6 +119,10 @@ class TranslateExecutablesPass
       passManager.addNestedPass<IREE::HAL::ExecutableVariantOp>(
           createTranslateTargetExecutableVariantsPass(targetName));
     }
+
+    IREE_TRACE_MESSAGE_DYNAMIC(INFO, executableOp.getSymName().str().data(),
+                               executableOp.getSymName().str().size());
```

Comment:
These are sort of redundant with the breadcrumbs - do you think it's worth keeping the messages too?